Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -801,14 +827,46 @@ public static void CreateUpdateRunSetFromJSON(RunsetExecutor runsetExecutor, Run
                                     }
 
-                                    if (customizedInputVar == null)
+                                    if (customizedInputVar != null)
+                                    {
+                                        customizedInputVar.DiffrentFromOrigin = true;
+                                        switch(inputValueConfig.VariableCustomizationType)
+                                        {
+                                            case InputValue.eVariableCustomizationType.Value:
+                                                customizedInputVar.VarValChanged = true;
+                                                customizedInputVar.Value = inputValueConfig.VariableCustomizedValue;
+                                                break;
+                                            case InputValue.eVariableCustomizationType.Variable://saving variable by Name- Legacy
+                                                customizedInputVar.MappedOutputType = VariableBase.eOutputType.Variable;
+                                                customizedInputVar.MappedOutputValue = inputValueConfig.VariableCustomizedValue;
+                                                break;
+                                            case InputValue.eVariableCustomizationType.OutputVariable:
+                                                customizedInputVar.MappedOutputType = VariableBase.eOutputType.OutputVariable;
+                                                customizedInputVar.MappedOutputValue = inputValueConfig.VariableCustomizedValue;
+                                                break;
+                                            case InputValue.eVariableCustomizationType.GlobalVariable:
+                                                customizedInputVar.MappedOutputType = VariableBase.eOutputType.GlobalVariable;
+                                                customizedInputVar.MappedOutputValue = inputValueConfig.VariableCustomizedValue;
+                                                break;    
+                                            case InputValue.eVariableCustomizationType.ApplicationModelParameter:
+                                                customizedInputVar.MappedOutputType = VariableBase.eOutputType.ApplicationModelParameter;
+                                                customizedInputVar.MappedOutputValue = inputValueConfig.VariableCustomizedValue;
+                                                break;
+                                            case InputValue.eVariableCustomizationType.DataSource:
+                                                customizedInputVar.MappedOutputType = VariableBase.eOutputType.DataSource;
+                                                customizedInputVar.MappedOutputValue = inputValueConfig.VariableCustomizedValue;
+                                                break;
+                                            default:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either merge this case with the identical one on line 839 or change one of the implementations.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5538205)